src/modules/networks/networks-store.ts

```diff
@@ -154,7 +154,7 @@ export class NetworksStore extends Store<State> {
     return this.updateNetworks();
   }
 
-  private async fetchNetworkByChainId(chainId: ChainId) {
+  async #fetchNetworkByChainId(chainId: ChainId) {
```

Comment:
why? :)

Reply:
Because I made a public method with the same name, so this was a way to resolve the name conflict